easybuild/framework/easyconfig/format/yeb.py

```diff
@@ -46,6 +46,7 @@
 YAML_DIR = r'%YAML'
 YAML_SEP = '---'
 YEB_FORMAT_EXTENSION = '.yeb'
+EB_FORMAT_EXTENSION = '.eb'
```

Comment:
this belongs in `format/one.py` since it has nothing to do with the `.yeb` format?